tests/ui/coroutine/gen_fn.e2024.stderr

```diff
@@ -0,0 +1,10 @@
+error: `gen` blocks are not yet implemented
```

Comment:
Shouldn't this say `gen` fns are not yet implemented?

Reply:
Yes 😅 there's a bunch of diagnostics bugs around `gen` blocks or functions, but i forgot to give this one a FIXME